### PR TITLE
feat:Add footer to CKB faucet.

### PR DESCRIPTION
**Pull request by IronLu233**
https://github.com/Magickbase/ckb-explorer-public-issues/issues/326

I compare the link target between [explorer](https://pudge.explorer.nervos.org/) and my code, they are correspond. 

Add a responsive footer
### Normal
image

### Small screen
image

### CODE REVIEW

File: src/sections/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+      <div className="flex flex-wrap flex-1 w-full max-w-[1086px]">
+        {renderFooterLinks(FOOTER_PLAIN_LINKS.foundation)}
+
+        <div className="flex w-full sm:w-auto sm:flex-1 sm:justify-center">
+          {renderFooterLinks(FOOTER_PLAIN_LINKS.developer)}
+        </div>
+
+        <div className="grid grid-rows-2 grid-cols-4 sm:grid-rows-3 sm:grid-cols-3 gap-x-10 gap-y-5 h-[148px] sm:w-[188px] sm:h-[214px]">
+          {FOOTER_SOCIAL_LINKS.map(({ title, icon: Icon, href }, index) => (
+            <a
```
Comment (Keith-CY) [inline src/sections/Footer.tsx]: External links should be opened in new tab/window

Comment (IronLu233) [reply src/sections/Footer.tsx]: updated. Say sorry that I check the link target but forget test the behavior of window.

File: src/sections/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+        href: "https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0002-ckb/0002-ckb.md",
+      },
+      {
+        title: "Faucet",
+        href: "https://faucet.nervos.org/",
+      },
+      {
+        title: "Submit Token Info",
+        href: "mailto:ckb-explorer@nervosnet.com?subject=Submit%20Token%20Info&body=Title:%20Submit%20Token%20Information%0a%0d----------%20Submit%20sUDT%20Token%20Information%20----------%0a%0dType%20Script:%0a%0d%20%20%20%20Code%20Hash:%0a%0d%20%20%20%20Hash%20Type:%0a%0d%20%20%20%20Args:%0a%0dInformation:%0a%0d%20%20%20Display%20Name:%0a%0d%20%20%20UAN:%0a%0d%20%20%20Decimal:%208%20(default)%0a%0d%20%20%20Description:%0a%0d%20%20%20Website:%0a%0d%20%20%20Icon%20File:%20attachment%20(40%20x%2040)%0a%0d%20%20%20Other%20Info:%0a%0d%0a%0dRef:%0a%0d1.%20UAN:%20https://github.com/nervosnetwork/rfcs/pull/335%0a%0d----------%20Submit%20NRC%20721%20Factory%20Information%20----------%0a%0dInformation:%0a%0d%20%20%20Deployment%20Tx%20Hash:%20%0a%0d%20%20%20Contract%20Source%20Code:%20%0a%0d%20%20%20Other%20Info:%0a%0d",
+      },
```
Comment (Keith-CY) [inline src/sections/Footer.tsx]: Unnecessary links on the faucet page, could be replaced by link to the explorer

File: src/sections/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+    <h4 className="font-medium text-[26px] leading-[30px] sm:text-2xl mb-6">
+      {title}
+    </h4>
+    <ul className="w-full">
+      {links.map(({ title, href }, index) => (
+        <li
+          key={index}
+          className="flex flex-col text-gray-400 hover:text-purple transition-all  mb-4"
+        >
+          <a href={href} target="_blank">
```
Comment (Keith-CY) [inline src/sections/Footer.tsx]: `rel` should be set to `noopener noreferrer`

Ref:
- https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/noopener
- https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/noreferrer
- https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cross-Origin-Opener-Policy

File: src/sections/Footer.tsx [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+        <div className="flex w-full sm:w-auto sm:flex-1 sm:justify-center">
+          {renderFooterLinks(FOOTER_PLAIN_LINKS.developer)}
+        </div>
+
+        <div className="grid grid-rows-2 grid-cols-4 sm:grid-rows-3 sm:grid-cols-3 gap-x-10 gap-y-5 h-[148px] sm:w-[188px] sm:h-[214px]">
+          {FOOTER_SOCIAL_LINKS.map(({ title, icon: Icon, href }, index) => (
+            <a
+              key={index}
+              href={href}
+              target="_blank"
```
Comment (Keith-CY) [inline src/sections/Footer.tsx]: Same as above

File: src/sections/Footer.tsx [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+              className="flex flex-col text-gray-75 hover:text-purple transition-all h-[58px] w-[36px] items-center"
+            >
+              <Icon />
+              <span className="mt-2 text-xs">{title}</span>
+            </a>
+          ))}
+        </div>
+      </div>
+      <p className="text-xs self-start leading-5 sm:leading-[14px] sm:self-center text-gray-400 text-center w-[200px] sm:w-fit">
+        Copyright © {year} Nervos Foundation.All Rights Reserved.
```
Comment (Keith-CY) [inline src/sections/Footer.tsx]:
```suggestion
        Copyright © {year} Nervos Foundation. All Rights Reserved.
```

File: src/sections/Footer.tsx (hunk 3 above)
Comment (IronLu233) [reply src/sections/Footer.tsx]: Updated. Learnt that without referrer it may take risk from the target link.

**Comment by IronLu233** [comment]
Seems I meet a bug in Github. Although I push my newer code to my fork, it is have much disparity from my local.

To resolve this problem, I open a new PR for it.

https://github.com/Magickbase/ckb-testnet-faucet-frontend/pull/9

Sorry for inconvenience.

**Pull request closed without merging**